e2e/uusi_hanke.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { test, expect } from '@playwright/test';
+import { perustaja, vastaava, suorittaja, rakennuttaja, asianhoitaja, testiData, testiOsoite, tarkistaTulokset } from './_setup';
```

Comment:
Should these unused `vastaava`, `asianhoitaja` and `tarkistaTulokset` imports be removed?